jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -257,6 +258,7 @@ YUI.add('deployment-flow', function() {
       if (this.refs.modelName) {
         modelName = this.refs.modelName.getValue();
       }
+      this.setState({deploying: true});
       this.props.deploy(
         this._handleClose, true, modelName, credential, cloud, region);
```

Comment:
Should we set deploying back to false in handleClose, just for proper cleaning up?